massgov/openmass
# DP-29680: Link nos per k to feedback in Views

**Description:**
See ticket

**Jira:** (Skip unless you are MA staff)
DP-29680


**To Test:**
Spot check the below views and any others that have a Nos per 1000 page views column. You can also grep the config for the old path as a secondary check.
`grep -A 15 -nr 'field: nos_per_1000_cleaned' conf/drupal/config/views.view.*.yml | grep "path: 'node/{{ nid }}/analytics'"`


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
